src/3d/qgs3dmapscene.cpp

```diff
@@ -320,6 +321,14 @@ QgsChunkedEntity::SceneState _sceneState( QgsCameraController *cameraController
 
 void Qgs3DMapScene::onCameraChanged()
 {
+  if ( mMap.projectionType() == Qt3DRender::QCameraLens::OrthographicProjection )
+  {
+    QRect viewportRect( QPoint( 0, 0 ), mEngine->size() );
+    const float viewWidthFromCenter = mEngine->camera()->position().distanceToPoint( mEngine->camera()->viewCenter() );
+    const float viewHeightFromCenter =  viewportRect.height() * viewWidthFromCenter / viewportRect.width();
+    mEngine->camera()->lens()->setOrthographicProjection( -viewWidthFromCenter, viewWidthFromCenter, -viewHeightFromCenter, viewHeightFromCenter, 1.0f, 10000.0f );
```

Comment:
I would suggest to keep the currently used near and far planes instead of setting range 1 to 1000. The rationale is that if we had objects in the scene further away (e.g. 2km), they may get clipped away by the far plane, which could then break the whole scene. (it's a bit delicate process where two things depend on each other - near/far planes are updated based on scene's content, and scene's content gets clipped based on near/far planes)